packages/extension-base/src/background/handlers/State.ts

```diff
@@ -256,13 +261,12 @@ export default class State {
 
       this.saveCurrentAuthList();
       this.updateDefaultAuthAccounts(authorizedAccounts);
-      delete this.#authRequests[id];
-      this.updateIconAuth();
+      clearAuth();
```

Comment:
Oh wow, where the auth requests never cleared upon completion? But how about the icon? I don't remember it danlling 🤔

Reply:
it did, I just extracted it to separate function, so that I can reuse it in `reject` callback